### DISCUSSION TITLE
Creating composite primary key

### DISCUSSION THREAD

**Discussion by DivyanshuBhoyar** [pocketbase/pocketbase#1089] [Q&A]
I want to create a many-to-many link table for group membership scenario,
what is the best way using pockebase ?
`group_id, user_id, created_at`
Using UI, can i create the composite primary key comprising of above two ids?

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand the question. Each record has a single primary key - `id` and this cannot change.

If you just want a unique constraint across multiple fields, then this cannot be done from the UI at the moment and you have to either create a [Go DB migration](https://pocketbase.io/docs/migrations/) OR manually add the unique index/constraint against your `pb_data/data.db`:

```sh
sqlite3 pb_data/data.db "CREATE UNIQUE INDEX your_index_name ON your_collection (group_id, user_id)"
```

**Comment by TaylorZun** [comment]
谢谢您的邮件。我已收到，会尽快阅读并给您回复。
祝你天天开心哦
